src/EditorFeatures/Core/Implementation/Outlining/RegionCollector.cs

```diff
@@ -30,14 +30,15 @@ internal class RegionCollector
         }
 
         public static void CollectOutliningSpans(
-            SyntacticDocument document,
+            Document document,
```

Comment:
Having a hard time figuring out what is still being used from this document. I seem some checks of the workspace kind...perhaps better to pass just `Workspace` if that's the case to make it clear other use of the document is evil?